source/configuring-ember/disabling-prototype-extensions.md

```diff
@@ -147,7 +147,7 @@ fullNameDidChange: observer('fullName', function() {
 })
 ```
 
-Evented functions are annotated using `Ember.on()`:
+Evented functions are annotated using [`Ember.on()`](https://emberjs.com/api/ember/2.15/namespaces/Ember/methods/on?anchor=on):
```

Comment:
Whoops, this seems to have gone the wrong way ... Mind fixing that? 😀

Reply:
This one doesn't exist on `release`